frappe/commands/utils.py

```diff
@@ -102,6 +103,7 @@ def clear_cache(context):
 			frappe.clear_cache()
 			clear_notifications()
 			clear_website_cache()
+			clear_translations()
```

Comment:
probably not required. `frappe.clear_cache()` clears EVERYTHING now. 

https://github.com/frappe/frappe/blob/7d4bdbebb9c80742e098b97bb1008fdb8c340fcc/frappe/__init__.py#L963-L964 


I got tired of weird cache_manager tracking and just added this nuclear option when no args are passed to frappe.clear_cache.

Reply:
Huh, I was under the impression that `bench clear-cache` didn't clear the translations cache. 🤔